openapi/index.yaml

```diff
@@ -416,85 +416,91 @@ parameters:
     x-example: DOWNLOAD
 definitions:
   ExtendedProfile:
-    $ref: "https://raw.githubusercontent.com/pagopa/io-functions-commons/v20.4.0/openapi/definitions.yaml#/ExtendedProfile"
+    $ref: "https://raw.githubusercontent.com/pagopa/io-functions-commons/v20.6.1/openapi/definitions.yaml#/ExtendedProfile"
```

Comment:
Pls upgrade all `io-functions-commons` references to `20.6.5` specification

Reply:
@gquadrati you did this, as said above, right?

Reply:
Yes, exactly